### DISCUSSION TITLE
React hook form Controller with Antd

### DISCUSSION THREAD

**Discussion by howdyAnkit** [react-hook-form/react-hook-form#6217] [Q&A]
I am using Antd  Fw multiselect (https://codesandbox.io/api/v1/sandboxes/define) with React hook form for Validation 
when clicking onChange the second value gets prefilled with API and I can see both the dropdowns filled with correct information

but when I check the Console details I find out the value for only Second dropdown is filled not the first one 
Please find below attached code below. I really tried each and every way possible

```tsx
<Controller
  name="Icecream"
  control={control}
  render={({ field }) => (
    <Select
    className="dropdown"
    style={{ width: 120 }}
    placeholder="Make"
    defaultValue={Model}
    onChange={handleProvinceChange}
    loading={dropdownLoader}
    {...field}
    options={options}
      />
    )}
/>
```

```tsx
<Controller
                  name="choclate"
                  render={({ field }) => (
                    <Select
                      className="dropdown"
                      style={{ width: 120 }}
                      placeholder="Model"
                      defaultValue={Model}
                      onChange={onSecondCityChange}
                      loading={dropdownLoader}
                      {...field}
                      options={modelOptions}
                    />
                  )}
                  control={control}
                />
```

**Comment by bluebill1049** [comment]
> https://codesandbox.io/api/v1/sandboxes/define

this link doesn't work

**Reply by howdyAnkit** [reply]
could you please select below attached image multselect dropdown from this link 
Thankyou for the apt response
(https://ant.design/components/select/)
![image](https://user-images.githubusercontent.com/40737659/129320050-7c4b5816-6113-46b4-b36e-444b694e1c51.png)

**Reply by howdyAnkit** [reply]
Hey, @bluebill1049  any update on this conversation did I missed something in the code/docs?

**Reply by Salmandabbakuti** [reply]
Could you share your working sandbox link again @howdyAnkit ?
